linera-execution/src/execution_state_actor.rs

```diff
@@ -7,7 +7,7 @@
 use std::sync::LazyLock;
 
 use custom_debug_derive::Debug;
-use futures::channel::mpsc;
+use futures::{channel::mpsc::{self, UnboundedReceiver}, StreamExt};
```

Comment:
You'll need to fix the formatting here for CI to pass:

```suggestion
use futures::{
    channel::mpsc::{self, UnboundedReceiver},
    StreamExt,
};
```